app/models/alchemy/page.rb

```diff
@@ -389,12 +404,43 @@ def update_node!(node)
       update_columns(hash)
     end
 
+    # Creates a new version
+    #
+    # And copy all current elements, if any exist.
+    #
+    # @return Alchemy::Version
+    #
+    def create_version
```

Comment:
Pass a `from` parameter in here. Optional, defaulting to the current version, so as an Admin, I could potentially decide whether to branch off the `public` or the `current` version.